src/ccontrol/UserQuerySelect.cc

```diff
@@ -424,13 +427,13 @@ void UserQuerySelect::setupMerger() {
 
     auto&& preFlightStmt = _qSession->getPreFlightStmt();
     if (preFlightStmt == nullptr) {
-        _qMetaUpdateStatus(qmeta::QInfo::FAILED, 0, 0);
+        _qMetaUpdateStatus(qmeta::QInfo::FAILED, 0, 0, 0);
         _errorExtra = "Could not create results table for query (no worker queries).";
         return;
     }
     if (not _infileMerger->makeResultsTableForQuery(*preFlightStmt)) {
         _errorExtra = _infileMerger->getError().getMsg();
-        _qMetaUpdateStatus(qmeta::QInfo::FAILED, 0, 0);
+        _qMetaUpdateStatus(qmeta::QInfo::FAILED, 0, 0, 0);
```

Comment:
You may avoid explicitly passing 3 zeroes here if you give the default values `=0` to the corresponding parameters of the method `_qMetaUpdateStatus`.